desc/continuation.py

```diff
@@ -105,6 +98,13 @@ def _solve_axisym(
             deltas = get_deltas({"surface": surf_i}, {"surface": surf_i2})
             surf_i = surf_i2
 
+        constraints_i = get_fixed_boundary_constraints(
```

Comment:
why was this moved into the loop? because now it matters which object is attacehd to the objectives, so we have to rebuild each time to ensure that the correct equilibrium's parameters are being changed when eqi.solve or perturb is called?

Reply:
yes exactly. Ideally we'll add back in the ability to "re-use" objectives that have already been compiled but this PR was getting long so figured that would wait for another one.